test/build/module-preload/snowpack.config.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
-  mount: {
-    public: '/',
-    './src': '/_dist_',
-  },
-  plugins: ['@snowpack/plugin-optimize'],
```

Comment:
This whole test was designed to test `{ preloadModules: true }`, which wasn’t even in the test! So it was deleted now that it’s covered by #1399.